pyproject.toml

```diff
@@ -1,73 +1,47 @@
-[project]
-name = "ReX-AI"
+[tool.poetry]
+name = "rex-ai"
```

Comment:
I noticed that the docs build URL uses "rex-xai" - can/should we keep to one or the other for consistency?

Reply:
@kellino what do you think about the naming - should we switch to use "rex-xai" everywhere?

Reply:
Yes, that's the most logical approach @liz-is. ReX-AI would imply that we are an AI framework of some sort, whereas we are an XAI (explainable AI) framework.